sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLCLIDriver.scala

```diff
@@ -585,7 +590,7 @@ private[hive] class SparkSQLCLIDriver extends CliDriver with Logging {
         if (insideSingleQuote || insideDoubleQuote) {
           // Ignores '/' in any case of quotes
         } else if (insideBracketedComment && line.charAt(index - 1) == '*' ) {
-          bracketedCommentLevel -= 1
+          bracketedCommentRightBound = index
```

Comment:
We need to keep `index` here? It seems we always decrement `bracketedCommentLevel` in the next loop, so we cannot use a boolean flag instead?

Reply:
I will try it.

Reply:
Thanks!